tests/protocol/test_extensions_goal_state.py

```diff
@@ -1,27 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
These 2 tests were moved to test_extensions_goal_state_from_extensions_config/from_vm_settings.py, along with the other parse tests